### ISSUE TITLE
sparql path expression with question mark

### ISSUE THREAD

**Issue by eroux** [blake-regalia/linked-data.syntaxes#15]
The following query is not highlighted correctly:

```sparql
select distinct ?hasParts ?nbVolumes where {
  ?R_RES a :Work .
  BIND(EXISTS{?R_RES :workLinkTo?/:workHasPart ?part} AS ?hasParts)
  optional {
    ?R_RES :workLinkTo?/:workLocation ?loc .
    FILTER( NOT EXISTS {?loc :workLocationEndVolume ?evol} )
    BIND(1 as ?nbVolumes)
  }
  optional {
    ?R_RES :workLinkTo?/:workLocation ?loc .
    ?loc :workLocationVolume ?bvol ;
         :workLocationEndVolume ?evol .
    BIND((?evol - ?bvol) as ?nbVolumes)
  }
  optional {
    ?R_RES :workLinkTo?/:workPartOf* ?rw .
    ?rw :workHasItem ?item .
    ?item a bdo:ItemImageAsset .
    ?item :itemVolumes ?nbVolumes .
    FILTER ( NOT EXISTS{?R_RES :workLinkTo?/:workHasPart ?part} )
  }
}
```

![Capture d’écran de 2019-10-22 17-00-30](https://user-images.githubusercontent.com/60868/67299926-89553500-f4ed-11e9-9041-a2ac7cd11235.png)

**Comment by blake-regalia** [comment]
Fixed in [v0.5.2](https://github.com/blake-regalia/linked-data.syntaxes/releases/tag/v0.5.2). Can be updated thru PackageControl.

**Comment by eroux** [comment]
excellent, thanks a lot!
